src/include/rados/rgw_file.h

```diff
@@ -39,6 +39,7 @@ enum rgw_fh_type {
   RGW_FS_TYPE_NIL = 0,
   RGW_FS_TYPE_FILE,
   RGW_FS_TYPE_DIRECTORY,
+  RGW_FS_TYPE_SYMBOLIC_LINK,
 };
 
 /*
```

Comment:
@taoCH #define LIBRGW_FILE_VER_EXTRA needs to increment as you're extending the interface